script.js

```diff
@@ -1,4 +1,123 @@
+// Global Variables
+// Track player 1 score as integer
+var intPlayerOneScore = 0;
+var intPlayerOneScoreAlt = 0;
+// Track player 1 score as string
+var playerOneScoreString = 0;
+var playerOneScoreAltString = 0;
+// Track player 2 score as integer
+var intPlayerTwoScore = 0;
+var intPlayerTwoScoreAlt = 0;
+// Track player 2 score as string
+var playerTwoScoreString = 0;
+var playerTwoScoreStringAlt = 0;
+// Tracking Final Score of Both Players
+var finalScorePlayerOne = "";
+var finalScorePlayerTwo = "";
+// gameTurns
+var gameTurn = "Player 1";
+
+//Create a diceRoll function
+var diceRoll = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  return randomInteger;
+};
+
+// Score comparison function
+var scoreComparison;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (gameTurn == `Comparison` && finalScorePlayerOne > finalScorePlayerTwo) {
+    return `Player 1 wins! `;
+  }
+  if (gameTurn == `Comparison` && finalScorePlayerOne < finalScorePlayerTwo) {
+    return `Player 2 wins! `;
+  }
+  // When player 1 hits submit
+  if (gameTurn == `Player 1` && input == ``) {
```

Comment:
This could be refactored into helper functions so that you dont have to repeat these lines of code again for player two